src/brsTypes/components/RoSGNode.ts

```diff
@@ -133,6 +133,7 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
             ifSGNodeFocus: [this.hasfocus, this.setfocus, this.isinfocuschain],
             ifSGNodeDict: [this.findnode, this.issamenode, this.subtype],
         });
+        this.appendMethod("callFunc", this.callFunc);
```

Comment:
I don't think this is documented anywhere but if you run `print findMemberFunction(node, "callfunc")` on RBI it will print `<Interface: ifSGNodeDict>`, can you add the method to that interface?

Reply:
Great find, can do!